terraform/create_tf_vars.sh

```diff
@@ -13,8 +13,18 @@ CF_PASSWORD_DEV=$(echo "$KEYS_JSON" | jq -r '.password')
 
 CF_SPACE="tanf-dev"
 
+S3_CREDENTIALS=$(cf service-key tdp-tf-states tdp-tf-key | tail -n +2)
```

Comment:
Don't need any of this any more either if we aren't using the aws_bucket variable.